project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 // The Play plugin
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.12")
 
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.4")
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.9")
```

Comment:
This PR explains this better than I can https://github.com/guardian/grid/pull/2517